src/User/UserServiceProvider.php

```diff
@@ -82,6 +86,14 @@ public function boot()
             return false;
         });
 
+        foreach ($this->app->make('flarum.user.groupProcessors') as $callback) {
```

Comment:
AFAIK, we use `snake_case` for multi-word services (at least in one place I found). :wink:

Reply:
@franzliedke I think I've seen snake case and dot notated in the code, we might want to work on truly standardizing that (like we are working towards with permissions)

Reply:
The dot notation is fine, I was only referring to the last part.